sswlib/src/main/java/common/Constants.java

```diff
@@ -29,7 +29,7 @@
 package common;
 
 public class Constants {
-    public final static String LibVersion = "0.0.1",
+    public final static String LibVersion = "0.7 Experimental 7",
```

Comment:
I believe that this needs to be "0.7.0" though the lib could be different than all the others.

Reply:
Should this also have a stable flag?

Reply:
Okay, I updated it to 0.7.0.

  I'm not sure the stable flag matters for a library.  Since the flag sin;t there already, I defaulted to not including it here.

Reply:
ok sounds good to me.